doc/documentation_data_format.rst

```diff
@@ -0,0 +1,689 @@
++-------------+-----------------+-------------------------+-------------+------------+--------------+----------+-----+
+
+*(wrapped for readability)*
+
++-----+---------------------------+---------------------------------+----------------------+----------------------------+
+| ... | [initializationPriorType] | [initializationPriorParameters] | [objectivePriorType] | [objectivePriorParameters] |
++=====+===========================+=================================+======================+============================+
+| ... | *see below*               | *see below*                     | *see below*          | *see below*                |
++-----+---------------------------+---------------------------------+----------------------+----------------------------+
+| ... | ...                       | ...                             | ...                  | ...                        |
++-----+---------------------------+---------------------------------+----------------------+----------------------------+
+
+Additional columns may be added.
+
+
+Detailed field description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``parameterId`` [STRING, NOT NULL]
+
+  The ``parameterId`` of the parameter described in this row. This has to match
+  the ID of a parameter specified in the SBML model, a parameter introduced
+  as override in the condition table, or a parameter occurring in the
+  ``observableParameters`` or ``noiseParameters`` column of the measurement table
+  (see above).
+
+- ``parameterName`` [STRING, OPTIONAL]
+
+  Parameter name to be used e.g. for plotting etc. Can be chosen freely. May
+  or may not coincide with the SBML parameter name.
+
+- ``parameterScale`` [lin|log|log10]
+
+  Scale of the parameter to be used during parameter estimation.
+
+- ``lowerBound`` [NUMERIC]
+
+  Lower bound of the parameter used for optimization.
+  Optional, if ``estimate==0``.
+  Must be provided in linear space, independent of ``parameterScale``.
+
+- ``upperBound`` [NUMERIC]
+
+  Upper bound of the parameter used for optimization.
+  Optional, if ``estimate==0``.
+  Must be provided in linear space, independent of ``parameterScale``.
+
+- ``nominalValue`` [NUMERIC]
+
+  Some parameter value to be used if
+  the parameter is not subject to estimation (see ``estimate`` below).
+  Must be provided in linear space, independent of ``parameterScale``.
+  Optional, unless ``estimate==0``.
+
+- ``estimate`` [BOOL 0|1]
+
+  1 or 0, depending on, if the parameter is estimated (1) or set to a fixed
+  value(0) (see ``nominalValue``).
+
+- ``initializationPriorType`` [STRING, OPTIONAL]
+
+  Prior types used for sampling of initial points for optimization. Sampled
+  points are clipped to lie inside the parameter boundaries specified by
+  ``lowerBound`` and ``upperBound``. Defaults to ``parameterScaleUniform``.
+
+  Possible prior types are:
+
+    - *uniform*: flat prior on linear parameters
```

Comment:
Nested list rendered as quote. Same for following list.

Reply:
not sure, seems fine on sphinx, only github does this ...

Reply:
okay, on checked there... would be good if it was working on both, as far as possible :)

Reply:
definitely. just not sure what's happening, it seems like valid rst. I think github doesn't have math/latex support yet, e.g.